pkg/internal/third_party/go-json-experiment/json/arshal_any.go

```diff
@@ -4,7 +4,10 @@
 
 package json
 
-import "reflect"
```

Comment:
@dsnet FYI Here and below is my very sad attempt at getting deterministic behavior. It's definitely not great performance-wise (see below), but I'm assuming a few of these arrays could be re-used via a pool.

```
> benchstat unsorted sorted
name                                                    old time/op    new time/op    delta
SwaggerSpec_ExperimentalMarshal/jsonv2_via_jsonv1-8       47.8ms ± 6%    51.7ms ± 8%   +8.21%  (p=0.001 n=9+10)
SwaggerSpec_ExperimentalMarshal/jsonv2-8                  31.7ms ± 4%    33.3ms ± 2%   +5.23%  (p=0.000 n=10+9)
SwaggerSpec_ExperimentalMarshal/jsonv2-canonicalized-8    49.9ms ± 4%    49.6ms ± 5%     ~     (p=0.481 n=10+10)

name                                                    old alloc/op   new alloc/op   delta
SwaggerSpec_ExperimentalMarshal/jsonv2_via_jsonv1-8       36.9MB ±11%    40.0MB ±12%     ~     (p=0.075 n=10+10)
SwaggerSpec_ExperimentalMarshal/jsonv2-8                  22.1MB ± 2%    24.7MB ± 2%  +11.78%  (p=0.000 n=10+10)
SwaggerSpec_ExperimentalMarshal/jsonv2-canonicalized-8    29.7MB ±16%    29.3MB ± 0%     ~     (p=0.220 n=10+6)

name                                                    old allocs/op  new allocs/op  delta
SwaggerSpec_ExperimentalMarshal/jsonv2_via_jsonv1-8        91.3k ± 0%    111.8k ± 0%  +22.40%  (p=0.000 n=10+10)
SwaggerSpec_ExperimentalMarshal/jsonv2-8                   91.3k ± 0%    111.8k ± 0%  +22.41%  (p=0.000 n=10+10)
SwaggerSpec_ExperimentalMarshal/jsonv2-canonicalized-8     92.9k ± 0%    113.4k ± 0%  +22.08%  (p=0.000 n=10+8)
```

Reply:
My apologies for the delay. I just merged https://github.com/go-json-experiment/json/commit/540f014424240312547dcccddf11a8229ca0f463, which adds `MarshalOptions.Deterministic`.

Reply:
Thank you so much! That's absolutely great